test/relativistic/vector/relativistic_velocity_orthogonal_to_movement_test.py

```diff
@@ -0,0 +1,81 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy.physics.units import speed_of_light
+from symplyphysics import (
+    units,
+    Quantity,
+    QuantityVector,
+    errors,
+)
+from symplyphysics.core.approx import assert_equal_vectors
+from symplyphysics.core.vectors.arithmetics import reject_cartesian_vector
+from symplyphysics.laws.relativistic.vector import relativistic_velocity_orthogonal_to_movement as law
+
+# Description
+## The velocity vector of the body in the proper frame of reference is (0.1*c, 0.2*c, -0.333*c). The velocity vector of the
+## proper frame of reference relative to the laboratory one is (-0.2*c, -0.4*c, 0). The velocity of the body in the lab frame
+## of reference is (-0.111*c, -0.222*c, -0.331*c).
+
+Args = namedtuple("Args", "up ul v")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    up = QuantityVector([Quantity(0.100 * speed_of_light), Quantity(0.200 * speed_of_light), Quantity(-0.333 * speed_of_light)])
+    ul = QuantityVector([Quantity(-0.111 * speed_of_light), Quantity(-0.222 * speed_of_light), Quantity(-0.331 * speed_of_light)])
+    v = QuantityVector([Quantity(-0.200 * speed_of_light), Quantity(-0.400 * speed_of_light), Quantity(0)])
+    return Args(up=up, ul=ul, v=v)
+
+
+def test_lab_law(test_args: Args) -> None:
+    result = law.calculate_orthogonal_velocity_component_in_lab_frame(test_args.up, test_args.v)
+    ul_orthogonal = reject_cartesian_vector(
+        test_args.ul.to_base_vector(),
+        test_args.v.to_base_vector()
+    )
+    assert_equal_vectors(
+        result,
+        QuantityVector.from_base_vector(ul_orthogonal),
+        tolerance=3e-3,
+        absolute_tolerance=1e-8,
```

Comment:
Don't use both tolerances. I will update code to throw, when both are passed.

Reply:
But this is compatible with how `pytest.assert` works. It accepts both values for tolerance and uses the absolute one if it detects zero, otherwise it defaults to the relative one. IMO it's the correct behavior. The problem is that some values in the vector might be zero and others are non-zero. So we need both of them to make it work.

Reply:
Approx is using the same approach and it led to hard to catch errors. I'll think how to make it better.

Reply:
I will accept it in this PR but I will ban both tolerances later. We will have option to use `relative_tolerance`, `absolute_tolerance` and `absolute_tolerance_if_zero`. The latter option will be possible to use together with `relative_tolerance`.

Reply:
OK, I'll take a look

Reply:
I'll take a look

Reply:
Okay, this sounds reasonable.